src/ansys/mechanical/core/run.py

```diff
@@ -111,6 +112,13 @@ def _cli_impl(
         if input_script:
             raise Exception("Cannot open in server mode with an input script.")
 
+    if not input_script and script_args:
+        raise Exception("Cannot add script arguments without an input script.")
+
+    if script_args:
+        if '"' in script_args:
```

Comment:
@klmcadams if I do this:

`ansys-mechanical -i script.py -script-args "arg1,arg2,arg3"`

then there are double quotes around the script args. Will this work? I think a lot of people are in the habit of putting double quotes around command line arguments that are long

Reply:
It will work! Whenever it reads the script-args string in, it removes the double quotes so it just becomes `arg1,arg2,arg3` which is why we have to add the double quotes around the args again whenever we append it to the args list. I updated the exception line to specify that you can't add double quotes around individual arguments within the --script-args string to clarify